internal/service/atlasuser/data_source_atlas_user_test.go

```diff
@@ -49,14 +49,18 @@ func TestAccConfigDSAtlasUser_ByUsername(t *testing.T) {
 			{
 				Config: testAccDSMongoDBAtlasUserByUsername(username),
 				Check: resource.ComposeTestCheckFunc(
-					dataSourceChecksForUser(dataSourceName, "", user)...,
+					dataSourceChecksForUser(dataSourceName, "", user, true)...,
 				),
 			},
 		},
 	})
 }
 
-func dataSourceChecksForUser(dataSourceName, attrPrefix string, user *admin.CloudAppUser) []resource.TestCheckFunc {
+func dataSourceChecksForUser(dataSourceName, attrPrefix string, user *admin.CloudAppUser, hasToCheckCountRoles bool) []resource.TestCheckFunc {
```

Comment:
@AgustinBettati you're right, changing to keep role count in singular, just check attribute exists in plural